spec/features/user_google_login_spec.rb

```diff
@@ -65,6 +65,10 @@
             continue.call(request)
           end
         end
+
+        spec.run
+
+        page.driver.browser.devtools.fetch.disable
```

Comment:
We were getting some flakiness (see below). Hopefully this will fix it. 🤞

```
Failures:

  1) Logging in as a User via Google auth when OmniAuth test mode is enabled and OmniAuth is mocked when there is no user in the databse with the email allows a user to sign up (and log in) with Google
     Failure/Error: continue.call(request)

     Selenium::WebDriver::Error::WebDriverError:
       -32602: Invalid InterceptionId.: 
     # ./spec/features/user_google_login_spec.rb:65:in `block (5 levels) in <top (required)>'

Finished in 24.27 seconds (files took 6.59 seconds to load)
27 examples, 1 failure

Failed examples:

rspec ./spec/features/user_google_login_spec.rb:30 # Logging in as a User via Google auth when OmniAuth test mode is enabled and OmniAuth is mocked when there is no user in the databse with the email allows a user to sign up (and log in) with Google
```